sanic/server.py

```diff
@@ -404,11 +407,12 @@ def serve_multiple(server_settings, workers, stop_event=None):
     server_settings['host'] = None
     server_settings['port'] = None
 
-    if stop_event is None:
-        stop_event = Event()
+    def sig_handler(signal, frame):
+        log.info("Received signal {}. Shutting down.".format(
```

Comment:
Without doing something here https://github.com/channelcat/sanic/pull/491/files#diff-bd84a6e523017a7f3d6bd72ee689dce2R414 you will get errors.